lib/services/ios-project-service.ts

```diff
@@ -194,6 +195,25 @@ class IOSProjectService implements  IPlatformProjectService {
         }).future<void>()();
     }
 
+	public getDebugOnDeviceSetup(): Mobile.IDebugOnDeviceSetup {
+		var tnsIosPackage = "";
+		if(options.frameworkPath) {
+			if(this.$fs.getFsStats(options.frameworkPath).wait().isFile()) {
+				this.$errors.fail("frameworkPath option must be path to directory which contains tns-ios framework");
+			}
+
+			tnsIosPackage = path.resolve(options.frameworkPath);
+		} else {
+			tnsIosPackage = this.$npm.install("tns-ios").wait();
```

Comment:
Not a merge stopper. This is a very lengthly operation. Can we warn the user to be patient?